Description: Removed obsolete embedded sys/capability.h header
 inclusion in mod_cap.c modules. Also it avoids building of the obsolete
 embedded libpcap.
Author: Francesco Paolo Lovergine <frankie@debian.org>
Forwarded: not needed
Bug-Debian: http://bugs.debian.org/479893

Index: proftpd-dfsg/lib/libcap/Makefile
===================================================================
--- proftpd-dfsg.orig/lib/libcap/Makefile	2018-01-14 23:22:58.000000000 +0100
+++ proftpd-dfsg/lib/libcap/Makefile	2018-01-14 23:22:58.000000000 +0100
@@ -23,7 +23,7 @@
 INCLS=libcap.h cap_names.h $(INCS)
 OBJS=$(addsuffix .o, $(FILES))
 
-all: $(LIBNAME)
+all: 
 
 _makenames: _makenames.c cap_names.sed
 	$(CC) $(CFLAGS) $(LDFLAGS) $< -o $@
Index: proftpd-dfsg/modules/mod_cap.c
===================================================================
--- proftpd-dfsg.orig/modules/mod_cap.c	2018-01-14 23:22:58.000000000 +0100
+++ proftpd-dfsg/modules/mod_cap.c	2018-01-14 23:22:58.000000000 +0100
@@ -42,10 +42,8 @@
 #  define _LINUX_BYTEORDER_GENERIC_H
 # endif
 
-# ifdef HAVE_LINUX_CAPABILITY_H
-#  include <linux/capability.h>
-# endif /* HAVE_LINUX_CAPABILITY_H */
-# include "../lib/libcap/include/sys/capability.h"
+#include <linux/capability.h>
+#include <sys/capability.h>
 
 /* What are these for? */
 # undef WNOHANG
